### PR TITLE
Small repairs

### PR DESCRIPTION
**Pull request by mvr320**
Already tested in our demo environment.

### CODE REVIEW

**Review by nickygerritsen — APPROVED** [review]

File: icpc-wf/ansible/files/grafana/my.cnf.j2 [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [client]
 user = collectdstatus
 password = {{ COLLECTD_MYSQL_PASSWORD }}
-host = {{ hostvars[host].ansible_host }}
+host = 127.0.0.1
```
Comment (meisterT) [inline icpc-wf/ansible/files/grafana/my.cnf.j2:4]: Was this change intentional?

Comment (mvr320) [reply icpc-wf/ansible/files/grafana/my.cnf.j2:4]: Yes, because we have 2 SQL servers we need to scrape from localhost and then export, instead of crawl from the monitoring server and then send to grafana on localhost.
